Foxtrot/Tests/ExtractorTest.cs

```diff
@@ -12,47 +12,262 @@
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CC1015.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: @"/optimize",
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CC1016.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+                yield return new Options(
+                    sourceFile: @"Foxtrot\Tests\CheckerTests\CheckerErrors.cs",
+                    foxtrotOptions: null,
+                    useContractReferenceAssemblies: true,
+                    compilerOptions: null,
+                    references: new[] { "mscorlib.dll", "System.dll", "ClousotTestHarness.dll" },
+                    libPaths: new[] { @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug" },
+                    compilerCode: "CS",
+                    useBinDir: false,
+                    useExe: false,
+                    mustSucceed: false);
+            }
+        }
+
+        public static IEnumerable<object> ExtractorTest
+        {
+            get
             {
-                try
-                {
-                    string sourceFile = (string)TestContext.DataRow["Name"];
-                    string options = (string)TestContext.DataRow["Options"];
-                    TestContext.WriteLine("FAILED: Name={0} Options={1}", sourceFile, options);
-                }
-                catch { }
+                return Enumerable.Range(0, ExtractorTestData.Count()).Select(i => new object[] { i });
```

Comment:
Is it necessary for xUnit to use `object` in this case? I know that with xUnit you can use `IEnumerable<int>` in this case.

And another question: why this test source returns a sequence of indices? Is it impossible to use `IEnumerable<TestCaseData>` where `TestCaseData` is an actual value at `ExtractorTestData`?